src/app/helpers/fields.js

```diff
@@ -0,0 +1,75 @@
+import { c } from 'ttag';
+
+export const getFields = () => [
+    { text: c('Contact field label').t`Name`, value: 'fn' },
```

Comment:
A global `const t = c('Contact field label').t` would not be simple to read?

Reply:
yeah, but we need the translations to be generated inside the components so that when the translation context changes, the strings are translated automatically without needing to reload the page